src/AccordionItem.svelte

```diff
@@ -27,12 +28,12 @@
 </script>
 
 <style>
-  li {
+  .default li {
```

Comment:
```suggestion
  li.default {
```

".default li" is unused